netlify.toml

```diff
@@ -1,2 +1,6 @@
+[build]
+  command = "npm run build"
```

Comment:
I don't think we need to do this even though it is in the starter now that the plugin has been updated. Just because I believe this is the default way to run when we have a package.json.

Reply:
Like it just kinda never mentions it? But I am here for the explicit, so I don't think it is worth worrying over! https://github.com/netlify/netlify-plugin-nextjs#installing-the-plugin